model/model.go

```diff
@@ -323,6 +326,13 @@ func (t TableLockType) String() string {
 	return ""
 }
 
+// FlashReplicaInfo means the flash replica info.
+type FlashReplicaInfo struct {
+	Count          uint64
+	LocationLabels []string
```

Comment:
Should we move the location labels out of the struct and put it in `TableInfo` as a sperate one. Because this one is also an essential part of isolate read.